.github/workflows/keyfactor-extension-release.yml

```diff
@@ -1,134 +0,0 @@
-            echo "Release Name: $($relName[-1])"
-            echo "RELEASE_NAME=$($relName[-1])" | Out-File $env:GITHUB_ENV -Encoding utf8 -Append
-            echo "Repo Name: $($repoName[-1])"
-            echo "REPO_NAME=$($repoName[-1])" | Out-File $env:GITHUB_ENV -Encoding utf8 -Append
-            echo "Release Version: $($relVersion[-1])"
-            echo "RELEASE_VERSION=$($relVersion[-1])" | Out-File $env:GITHUB_ENV -Encoding utf8 -Append
-        
-      - uses: actions/setup-dotnet@v1
-        with:
-            dotnet-version: '3.1.x' # SDK Version to use; x will use the latest version of the 3.1 channel      
-            #dotnet-version: 
-        
-      - name: Add Package Source
-        run: |
-          dotnet nuget add source https://nuget.pkg.github.com/Keyfactor/index.json -n github -u ${{ github.actor }} -p ${{ secrets.BUILD_PACKAGE_ACCESS }} --store-password-in-clear-text
-      
-      # Configures msbuild path envrionment
-      - name: setup-msbuild
-        uses: microsoft/setup-msbuild@v1
-      
-      # Restores Packages to Local Machine
-      - name: restore nuget packages
-        run: |
-          nuget restore ${{ env.SOLUTION_PATH  }}
-          
-      - name: Create Release
-        id: create_release
-        #uses: zendesk/action-create-release@v1
-        uses: keyfactor/action-create-release@786b73035fa09790f9eb11bb86834a6d7af1c256
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          release_name: ${{ env.REPO_NAME }} ${{ env.RELEASE_VERSION }}
-          body: |
-            [Changelog](../CHANGELOG.MD)
-          draft: false
-          prerelease: false
-          auto_increment_type: patch
-          tag_schema: semantic
-          commitish: ${{ github.sha }}          
-      
-      # update version number of AssemblyInfo.cs file
-      - name: Increment Assembly Version
-        run: |
-            $VersionRegex = "\d+\.\d+\.\d+"
-            $assemblyInfoFiles = (Get-ChildItem -Include AssemblyInfo.cs -File -Recurse).fullname
-            $newVer = "${{ steps.create_release.outputs.current_tag }}".TrimStart('v').Split('-')[0]
-            echo "Prepared to overwrite Assembly version to: ${newVer}"
-            foreach ($assemblyInfoFile in $assemblyInfoFiles)
-            {
-              $filecontent = Get-Content($assemblyInfoFile)
-              attrib $assemblyInfoFile -r
-              $filecontent -replace $VersionRegex, $newVer | Out-File $assemblyInfoFile
-            }
-      
-      - name: Execute MSBuild Commands
-        run: |
-          MSBuild.exe $Env:SOLUTION_PATH -p:RestorePackagesConfig=false -p:Configuration=Release       
-      
-      - name: Archive Files
-        if: ${{ success() }}
-        run: |
-           md ${{ github.workspace }}\zip\Keyfactor
-           Compress-Archive -Path `
-           ${{ env.SOLUTION_FOLDER }}\${{ env.PROJECT_FOLDER }}\bin\Release\netcoreapp3.1\* `
```

Comment:
This is the relevant line that would package the build output in the old workflow @fiddlermikey

Reply:
Updated the output folder